engine/collection/compliance/engine.go

```diff
@@ -195,6 +198,11 @@ func (e *Engine) Ready() <-chan struct{} {
 	}
 	e.lm.OnStart(func() {
 		e.unit.Launch(e.loop)
```

Comment:
Would be great to also update the Collectors' Compliance Engine to the new `ComponentManager` ... but lets do that in a separate PR

Reply:
I wanted to do that but:
1) It increases scope of PR a lot
2) We depend on some unit functionality which is not present in component manager.